content/chronograf/v1.9/guides/dashboard-template-variables.md

```diff
@@ -144,12 +188,65 @@ In cell queries, it should be used in the `GROUP BY time()` clause that accompan
 SELECT mean("usage_system") AS "Average System CPU Usage"
 FROM "telegraf".."cpu"
 WHERE time > :dashboardtime:
-GROUP BY time(:interval:)
+GROUP BY time(:autoInterval:)
+```
+{{% /tab-content %}}
+{{% tab-content %}}
+- [`v.timeRangeStart`](#vtimerangestart)
+- [`v.timeRangeStop`](#vtimerangestop)
+- [`v.windowPeriod`](#vwindowperiod)
+
+### v.timeRangeStart
+
+The `v.timeRangeStart` template variable is controlled by the "time" dropdown in your Chronograf dashboard.
+
+<img src="/img/chronograf/1-6-template-vars-time-dropdown.png" style="width:100%;max-width:549px;" alt="Dashboard time selector"/>
+
+If using relative times, it represents the time offset specified in the dropdown (-5m, -15m, -30m, etc.) and assumes time is relative to "now".
+If using absolute times defined by the date picker, `v.timeRangeStart` is populated with lower threshold.
+
+```js
+from(bucket: "telegraf/autogen")
+  |> range(start: v.timeRangeStart)
+  |> filter(fn: (r) => r._measurement == "cpu" and r._field == "usage_system")
 ```
 
+{{% note %}}
+To use the date picker to specify a particular time range in the past
+which does not include "now", use `v.timeRangeStart` as the lower limit and
+[`v.timeRangeStop`](#vtimerangestop) as the upper limit.
+{{% /note %}}
+
+### v.timeRangeStop
+The `v.timeRangeStop` template variable is defined by the upper time limit specified using the date picker.
+
+<img src="/img/chronograf/1-6-template-vars-date-picker.png" style="width:100%;max-width:762px;" alt="Dashboard date picker"/>
+
+It will inherit `now()` when using relative time frames or the upper time limit when using absolute time frames.
+
+```js
+from(bucket: "telegraf/autogen")
+  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)
+  |> filter(fn: (r) => r._measurement == "cpu" and r._field == "usage_system")
+```
+
+### v.windowPeriod
+The `v.windowPeriod` template variable is controlled by the display width of the
+dashboard cell and is calculated by the duration of time that each pixel covers.
+Use the `v.windowPeriod` variable to downsample data to use only enough points to
+display one point per pixel.
```

Comment:
To be sure, is this the sole use case?

Reply:
Is this accurate:
Use the `v.windowPeriod` variable to limit downsample data to display a maximum of one point per pixel.